CODE_OF_CONDUCT.md

```diff
@@ -4,9 +4,9 @@
 
 In the interest of fostering an open and welcoming environment, we as
 contributors and maintainers pledge to making participation in our project and
-our community a harassment-free experience for everyone, regardless of age, body
+community a harassment-free experience for everyone, regardless of age, body
```

Comment:
Please don't update this file, it's based on a specific template.

Reply:
Sorry, I'll revert the changes made.